schematics/nav/files/__path__/__name@dasherize@if-flat__/__name@dasherize__.component.spec.ts

```diff
@@ -0,0 +1,26 @@
+
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+
+import { <%= classify(name) %>Component } from './<%= dasherize(name) %>.component';
+
+describe('<%= classify(name) %>Component', () => {
+  let component: <%= classify(name) %>Component;
+  let fixture: ComponentFixture<<%= classify(name) %>Component>;
+
+  beforeEach(async(() => {
```

Comment:
Use `fakeAsync` instead of `async`.

Reply:
This code was copied from the base create component schematic.